CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
+
+### Bug Fixes
+
+* **ecr-assets:** fix assembly creation to not include absolute paths to docker files ([#5836](https://github.com/aws/aws-cdk/pull/5836)) ([6bc8ecc](https://github.com/aws/aws-cdk/pull/5833/commits/6bc8eccdba21a10944c81ababe07df37c07481d0)), fixes ([#5807](https://github.com/aws/aws-cdk/issues/5807))
```

Comment:
Use the same format as other change log entries... I don't think the "fixes" is necessary

Reply:
Its kind of incosistent. Some do have "closes..". I think its valuable to link to the issue itself as it provides the details for the actual problem.

Reply:
Ohh. Okay then, disregard!